packages/typescript-resolver-files/src/addVirtualTypesFileToTsMorphProject/addVirtualTypesFileToTsMorphProject.ts

```diff
@@ -69,16 +69,19 @@ const generateVirtualTypesFile = async ({
   const addResultAsComplextOutput =
     convertPluginOutputToComplextPluginOutput(addResult);
 
+  const print = (value: string[] | undefined): string =>
+    value ? value.join('\n') : '';
+
   return {
     filePath: resolverTypesPath,
     content: `
-    ${addResultAsComplextOutput.prepend?.join('\n')}
-    ${typescriptResult.prepend?.join('\n')}
-    ${typescriptResolversResult.prepend?.join('\n')}
+    ${print(addResultAsComplextOutput.prepend)}
```

Comment:
This was previously printing `undefined` literal string into the generated template.